src/clr-angular/progress/progress-bars/progress-bar.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input, HostBinding } from '@angular/core';
+
+@Component({
+  selector: 'clr-progress-bar',
+  template: `
+  <ng-container>
```

Comment:
Container?

Reply:
It's React habit. Did not even cross my mind that I don't need it.